.github/workflows/build-test.yml

```diff
@@ -97,7 +46,7 @@ jobs:
       - name: '[Prep 4] Set version'
         id: version
         run: |
-            echo "version=$(cat manifest.yml | grep version | cut -f 2 -d: | awk '{$1=$1};1')" >> $GITHUB_OUTPUT 
+            echo "version=$(cat manifest.yaml | grep version | cut -f 2 -d: | awk '{$1=$1};1')" >> $GITHUB_OUTPUT 
```

Comment:
Seems the branch is always a word `HEAD`:

```yaml
build:
  branch: HEAD
  timestamp: 1750094935510
  commit: 484d8ed94d04a7c09870ea3e83cb1a3266a23805
```

Reply:
Fixed in the latest commit. The `embed-metadata` action can't auto-detect branches from `pull_request` events, so the calling workflow needs to tell the action what to use.